Felienne/hedy
# [UI IMPROVEMENT] No longer shows success message when empty output warning

**Description**
In this PR we fix a small QoL feature so we no longer show a success message in the case of an empty output. This can, for example, happen in level 1 when we only use an `ask` but without any `print` or `echo`. There is already a corresponding warning, but as this is front-end calculated this is not caught by the `response` check and the success message is still shown. This is fixed.

**Fixes < issue or discussion number >**
This PR fixes #2298

**How to test**
Navigate to level 1 and verify that the following code does give a warning but not a success message:
```
ask How do you do?
```
